network/stellar-disbursement-platform/admin-guide/configuring-sdp.mdx

```diff
@@ -0,0 +1,187 @@
+
+### Messaging Configuration
+
+Messaging Configuration allows configuring the messaging service used to send messages to recipients and sdp dashboard users.
+* `EMAIL_SENDER_TYPE`: The messenger type used to send invitations to new dashboard users. Options: "DRY_RUN", "AWS_EMAIL". Default: "DRY_RUN".
+* `SMS_SENDER_TYPE`: The messenger type used to send SMS messages to recipients. Options: "DRY_RUN", "TWILIO_SMS", "AWS_SMS". Default: "DRY_RUN".
+
+#### AWS Configuration
+The following configurations are required when using AWS SES or SNS to send emails or SMS messages.
+* `AWS_ACCESS_KEY_ID` - *[secret]* The AWS access key ID.
+* `AWS_REGION` - The AWS region where the SES service is available.
+* `AWS_SECRET_ACCESS_KEY` - *[secret]* The AWS secret access key.
+* `AWS_SES_SENDER_ID` - The email that AWS SES will use as the sender when sending emails. Required when `EMAIL_SENDER_TYPE` is set to "AWS_EMAIL".
+* `AWS_SNS_SENDER_ID` - The sender ID to use when sending SMS messages using AWS SNS. Required when `SMS_SENDER_TYPE` is set to "AWS_SMS".
+
+#### Twilio Configuration
+The following configurations are required when `SMS_SENDER_TYPE` is set to "TWILIO_SMS".
+* `TWILIO_ACCOUNT_SID` - *[secret]* The Twilio account SID.
+* `TWILIO_AUTH_TOKEN` - *[secret]* The Twilio auth token.
+* `TWILIO_SERVICE_SID` - The Twilio service SID.
+
+General Messaging Configuration:
+* `MAX_INVITATION_SMS_RESEND_ATTEMPTS` - The maximum number of attempts to resend the SMS invitation to the Receiver Wallets. Default: 3.
+
+### Stellar Configuration
+Stellar Configuration allows configuring accounts, transactions, and other Stellar-related settings.
+
+* `NETWORK_PASSPHRASE` - The Stellar network passphrase. Default "Test SDF Network ; September 2015".
+* `HORIZON_URL` - The URL of the Horizon server to use for submitting transactions. Default "https://horizon-testnet.stellar.org/".
+* `SEP10_SIGNING_PUBLIC_KEY` - The public key of the Stellar account that signs the SEP-10 transactions. It's also used to sign URLs.
+* `SEP10_SIGNING_PRIVATE_KEY` - *[secret]* The private key of the Stellar account that signs the SEP-10 transactions. It's also used to sign URLs.
+* `MAX_BASE_FEE` - The max base fee for submitting a Stellar transaction. Default: 10000.
+
+The remaining configurations related to distribution accounts are detailed in the `Stellar accounts configuration` section of [1.x to 2.x Migration Guide](single-tenant-to-multi-tenant-migration#environment-variables).
+
+### Security Configuration
+Security Configuration allows configuring the security aspects of the SDP Core Service.
+
+* `CORS_ALLOWED_ORIGINS` - Specifies the domains allowed to make cross-origin requests. "*" means all domains are allowed. Domains can contain wildcards, e.g., "https://*.example.com".
+* `SEP24_JWT_SECRET` - *[secret]* The secret used to sign the JWT token for SEP-24 transactions. This secret is used during the receiver wallet registration flow.
+
+#### Authentication Configuration
```

Comment:
wondering if we should call this Dashboard Authentication Configuration

Reply:
Addressed in e015ac3772fccc33777dbb49b4c5474f46f86ae5